tests/test_membership.py

```diff
@@ -0,0 +1,171 @@
+# Copyright 2018 Comcast Cable Communications Management, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""TODO: Add module docstring."""
+import datetime
+
+import responses
+
+from vinyldns.membership import Group, GroupChange, ListAdminsResponse, ListGroupsResponse, ListGroupChangesResponse, \
+    ListMembersResponse, Member, User
+from vinyldns.serdes import to_json_string, from_json_string
+from sampledata import sample_group, sample_group2
+
+
+def check_groups_are_same(a, b):
+    if a is None and b is None:
+        assert a == b
+    else:
+        assert a.id == b.id
+        assert a.description == b.description
```

Comment:
Think we need a check for `email` and maybe `status` here as well.

Reply:
I can add more checks

Reply:
actually just email